code/modules/antagonists/traitor/objectives/smuggling.dm

```diff
@@ -0,0 +1,122 @@
+/datum/traitor_objective_category/smuggle
+	name = "Smuggling"
+	objectives = list(
+		/datum/traitor_objective/smuggle = 1,
+	)
+
+///smuggle! bring a traitor item from its arrival area to the cargo shuttle, where the objective completes on selling the item
+/datum/traitor_objective/smuggle
+	name = "Smuggle \[CONTRABAND] from \[AREA] off the station via cargo shuttle."
+	description = "Go to a designated area, pick up syndicate contraband, and get it off the station via the cargo shuttle. \
+	You will instantly fail this objective if anyone else picks up your contraband. If you fail, you are liable for the costs \
```

Comment:
This seems like a fairly unintuitive way to do this. You can't smuggle it in by giving it to cargo wrapped, you can't smuggle it by pretending it's a cube, you can't smuggle it if they happen to sort through the cargo loaded aboard. The practical method to accomplishing this task is ensuring no living hands will be able to touch your smuggled goods, really amounts to just killing all cargo members and launching ASAP. That's lame and really doesn't allow for any kind of deception at all.

You just need to have the smuggled goods self delete after a set amount of time and have the objective fail if it deletes in any instance that isn't through delivery. That way, this is more about being quick with the objective, and allows you to maybe complete the objective by smuggling it along with cargo bounties or some such, which cargo is incentivized to do quickly.